pygeoapi/pygeoapi.config.yml

```diff
@@ -299,11 +299,15 @@ resources:
               subjectOf:
                   "@id": schema:subjectOf
                   "@type": "@id"
+            - hyf: https://www.opengis.net/def/schema/hy_features/hyf
+              mainstem_uri:
+                "@id": hyf:flowpath
```

Comment:
This is a bit of a stretch, but I think this makes the most sense. 
![image](https://user-images.githubusercontent.com/1492803/156847708-1817d78c-479e-418b-9c86-59158223cc9d.png)

We are using the flowpath relation from hydronetwork to HY_Flowpath. The other relation that could be used in linear element, but I think it's just as indirect and less semantically useful.

Reply:
A couple other ways to conceptualize this. 

We could say that ref_gages are HY_Hydrolocation features of type hydrometricStation. In which case, we would link to the mainstem as the "linearElement" of the HY_Hydrolocation's indirect position. 

The other association would be to skip the indirect position concepts and just use sosa:hasUltimateFeatureofInterest. 

My knee jerk is to use sosa for mainstems and linearelement for the actual reachcode/measure indirect position. Thoughts??